spec/factories/dialog_field.rb

```diff
@@ -1,6 +1,7 @@
 FactoryGirl.define do
   factory :dialog_field do
-    name "Dialog Field"
+    sequence(:name) { |n| "Dialog Field #{seq_padded_for_sorting(n)}" }
+    sequence(:label) { |n| "Dialog Field label #{seq_padded_for_sorting(n)}" }
```

Comment:
AFAICT there's no validation on label at all - so wondering if you can get away with not providing a default value? I think the label could be omitted from tests that aren't explicitly dealing with label behavior. If they are, you probably will want to create the label in the test body, rather than rely on the default value here.